### PR TITLE
Weights to tensor

### PR DESCRIPTION
**Pull request by Semyon1104**

### CODE REVIEW

File: docs/json_for_test/invalid-[.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (aobolensk) [inline docs/json_for_test/invalid-[.json:1]: Actually, there is no need to check syntactically invalid jsons (json parsing library already does that well). I suggest to check semantically invalid json (json that contains unexpected values and does not describe layer structure)

**Review by aobolensk — CHANGES_REQUESTED** [review]

File: docs/json_for_test/invalid-[.json (hunk 1 above)
Comment (Semyon1104) [reply docs/json_for_test/invalid-[.json:1]: Yes, I understand, I'm working on it

**Comment by aobolensk** [comment]
Why do we have 197k lines?

**Comment by Semyon1104** [comment]
> Why do we have 197k lines?

Because I added the json file with the model weights to the docs folder

File: src/Weights_Reader/reader_weights.cpp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+void parse_json_shape(const json& j, std::vector<size_t>& shape, size_t dim) {
+  if (dim == 0) {
+    if (j.is_array()) {
+      if (j.empty()) {
+        shape.push_back(0);
+        return;
+      }
+      parse_json_shape(j.front(), shape, dim + 1);
+    } else {
+      shape.push_back(0);
```
Comment (aobolensk) [inline src/Weights_Reader/reader_weights.cpp:69]: This line is worth to be covered

**Review by aobolensk — APPROVED** [review]

File: test/model_read/json_for_test/invalid-}}.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "layer1": [1.0, 2.0, "three", 4.0, 5.0],
+  "layer2": [
+    [1.1, 2.1, "three.point.one", 4.1, 5.1],
+    [6.1, 7.1, 8.1, 9.1, 10.1]
+  ],
```
Comment (aobolensk) [inline test/model_read/json_for_test/invalid-}}.json:6]: Need to remove unused invalid json files

**Pull request merged**